src/_snadra/app.py

```diff
@@ -47,8 +48,11 @@ def _setup_prompt(self) -> None:  # pragma: no cover
         the `sys.stdout` and `sys.stderr` which disturbes `pytest`.
         """
         self.__prompt: "PromptSession[str]" = PromptSession(
-            "snadra > ",
+            f"({state['current_workspace']}) snadra > ",
             auto_suggest=AutoSuggestFromHistory(),
             history=InMemoryHistory(),
             completer=WordCompleter(self.commands.keywords),
         )
+
+    def update_active_workspace(self) -> None:
```

Comment:
Where this gets called?

Reply:
this needs to be called when you update the state